### DISCUSSION TITLE
Quarto blocks links in SVG files

### DISCUSSION THREAD

**Discussion by fkgruber** [quarto-dev/quarto-cli#8952] [Q&A]
### Bug description

if I add an svg image with an embedded link the link does not work in rendered html or confluence

### Steps to reproduce

Why does quarto remove links from SVG files? When rmardown renders the same file to html it respects the links in svg. 

 For example, I used the following plantuml to generate a diagram with a link in svg.
```planuml
@startuml
start
[[http://plantuml.com]]:Some activity;
:Some [[http://plantuml.com link]];
end
@enduml
```

` ![img](test_link.svg)`

then I use

```shell
quarto render test_link.qmd
```

and

```R
rmarkdown::render("test_link.qmd")
```

The rmarkdown html has an activate link in the images while the quarto does not. Is there an option in quarto to allow for the links?

thanks
 FKG

### Expected behavior

link in image should be clickable as in rmarkdown html 

### Actual behavior

can't click on link

### Quarto check output

Quarto 1.4.514
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.9: OK
      Dart Sass version 1.69.5: OK
      Deno version 1.37.2: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.4.514
      Path: /Users/user/miniconda3/lib/python3.11/site-packages/quarto_cli/quarto-1.4.514/bin

[✓] Checking tools....................OK
      TinyTeX: (not installed)
      Chromium: (not installed)

[✓] Checking LaTeX....................OK
      Using: Installation From Path
      Path: /Library/TeX/texbin
      Version: 2023

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.11.5 (Conda)
      Path: /Users/user/miniconda3/bin/python
      Jupyter: 5.5.0
      Kernels: python3

[✓] Checking Jupyter engine render....OK

[✓] Checking R installation...........OK
      Version: 4.3.2
      Path: /Library/Frameworks/R.framework/Resources
      LibPaths:
        - /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library
      knitr: 1.45
      rmarkdown: 2.25

[✓] Checking Knitr engine render......OK

**Comment by cderv** [comment]
Can you format your example correctly so that we can see the code and try to reproduce it? 

See guide at https://quarto.org/bug-reports.html for correct formatting

Thanks

**Comment by fkgruber** [comment]
I got it working by adding -M embed-resources:true

**Reply by fkgruber** [reply]
by the way is this the expected behavior? If you don't use embed-resources then the links inside svg are suppose to break?

**Comment by fkgruber** [comment]
![test_link](https://github.com/quarto-dev/quarto-cli/assets/4311763/55cd1f18-f854-494c-ad38-cbb0a7d4e78e)
I was using
`````
---
# YAML header created by ox-ravel
format:
  html:
    toc: true
title: quarto svg links
author: Fred K. Gruber
---
Why does quarto remove links from SVG files? When rmardown renders the same file to html it respects the links in svg
```
@startgantt
[task1] requires 10 days
[task1] links to [[http://plantuml.com mylink]]
@endgantt
```
![](test_link.svg)

`````

**Reply by cderv** [reply]
Is this the full .qmd ? I don't have `test_link.svg` available, and the code is just a regular code block so it won't make a SVG. 

Please do send a full example we can look at. We won't be able to answer the question otherwise

**Comment by mcanouil** [comment]
@fkgruber Next time, please follow-up on the already opened discussion rather than opening an issue.

Closing this as outdated:
- #8958
